File: sunbin/day5/python/003_중복된숫자개수.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120583

def solution(array, n):
    answer = array.count(n)
    return answer

# 다른 풀이
def solution_b(array, n):
  new_array = []
  for i in array:
    if i == n:
      new_array.append(i)

solution([1, 1, 2, 3, 4, 5], 1)
solution([2, 3, 4], 1)

File: sunbin/day5/003_중복된숫자개수.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120583

function solution(array, n) {
  const newArray = [];
  
  array.forEach(el => {
      if (el === n) {
          newArray.push(el);
      };
  });
  return newArray.length
}

// 다른풀이: filter
function solutionB(array, n) {
  const answer = 0;
  let newArray = array.filter((item) => itme === n);
  answer = newArray.length;

  return answer
}